agent/test/spark/app/appmap.yml

```
@@ -1,4 +1,4 @@
 name: sparkexample
-appmap_dir: app/build/tmp/appmap
```

This error was masked by the problem handling `appmap.output.directory`.